source/guides/references/changelog.md

```diff
@@ -2,6 +2,29 @@
 title: Changelog
 ---
 
+## 3.1.0
+
+**Summary:**
+
+- Cypress can now run recorded tests in parallel across multiple virtual machines. Fixes {% issue 1690 %} and {% issue 64 %}.
+
+**Features:**
+
+- `cypress run` accept a new `--parallel` flag to run specs in parallel.
+- `cypress run` accepts a new `--group` flag to group multiple spec files within a single run. Fixes {% issue 2153 %}
+- `cypress run` accepts a new `--ci-build-id` flag.
+- The Dashboard Service has new "Insights" available within each run. You can now see how the run was parallelized and calculate how adding or removing virtual machines would change future runtime.
```

Comment:
This all sounds most excellent! Looking forward to it: good work Team Cypress! 👍